### ISSUE TITLE
Typo in collections documentation

### ISSUE THREAD

**Issue by shamrin** [thefrontside/effection#653]
Typo:

https://github.com/thefrontside/effection/blob/b904213de729a2964deb9d4c07dbce2d01f2bb50/website/docs/guides/collections.md?plain=1#L169

Should it be something like this instead?

```js
   channel.send({ type: 'person', name: 'Jeff' }); // our filtered stream skips over this value 
```

**Comment by jnicklas** [comment]
Yes, that looks like a typo.

**Comment by cowboyd** [comment]
closed by #655
